core/templates/dev/head/domain/utilities/StatusObjectFactory.js

```diff
@@ -0,0 +1,32 @@
+// Copyright 2015 The Oppia Authors. All Rights Reserved.
```

Comment:
I suspect this file might be better placed in ``domain/status`` instead of utilities. Let's try and avoid proliferating the "dumping ground" pattern utilities can easily become.